test/e2e/util/BUILD

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["nodes_util.go"],
+    importpath = "k8s.io/kubernetes/test/e2e/util",
```

Comment:
Please create OWNERS for this new directory

Reply:
@fejta Sure. Taking care of it right-away.

Reply:
I'm using the same list from [test/e2e/framework](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/OWNERS) for this. Please let me know if that needs to be changed.